modules/tile-converter/src/i3s-converter/helpers/geometry-attributes.ts

```diff
@@ -212,6 +212,10 @@ function unifyObjectsByFeatureId(
         existedObject.texCoords,
         currentObject.texCoords
       );
+      existedObject.uvRegions = concatenateTypedArrays(
```

Comment:
Existed sounds strange, it is not a noun (it is past tense of a verb) . Not quite sure what it is trying to say but I would recommend that you make another effort to name this variable.
```suggestion
      existingObject.uvRegions = concatenateTypedArrays(
```

Reply:
Maybe a loop could be considered here?
```
for (const attributeName in ['positions', 'normals', ..., 'uvRegions']) {
  existing object[attributeName] = concatenateTypedArrays(...
```